deploy/template.yaml

```diff
@@ -174,6 +179,29 @@ Resources:
                           Fn::ImportValue:
                             Fn::Sub: "${VpcStackName}-ProtectedSubnetIdB"
 
+  ResultsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: Private
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+      BucketName: !Sub ${AWS::StackName}-test-results
+      LifecycleConfiguration:
+        Rules:
+          - ExpirationInDays: 1825 # Five years
```

Comment:
This feels like a very long time, can you check the policy with Ben/Security?

Reply:
Reduced to 365 days